docs/_functions/domain/SRV.md

```diff
@@ -7,6 +7,9 @@ parameters:
   - port
   - target
   - modifiers...
+parameter_types:
+  weight: number
+  port: number
```

Comment:
Let's face it, I'm not familiar with TypeScript typings (configurations). So I was wondering, when do you include which `parameter_types`? I feel like I'm missing `priority` here?

```suggestion
parameter_types:
  priority: number
  weight: number
  port: number
```

Reply:
I set a bunch of defaults here: https://github.com/StackExchange/dnscontrol/pull/1830/files#diff-76e760c4509933f6592ab7a1e50b4934736913b3cdfd91c00e1232ff33a81e82R52-R62. That said, it might make sense to get rid of that and explicitly specify all the parameter types for every function. Thoughts?

Reply:
Aha! I think I personally would opt for the duplication;
1. Everything in one place
2. No discussion when something is or isn't 'defaults'.
3. No need to look back on a PR if the missing param happens to be defined as a default.
4. Defaults could be left behind during refactoring activities.